src/ledger/LedgerStateAccountSQL.cpp

```diff
@@ -229,8 +229,11 @@ LedgerState::storeSignersInDatabase(StateEntry const& state)
     std::vector<Signer> signers;
     if (previousEntry)
     {
-        auto key = LedgerEntryKey(*previousEntry);
-        signers = loadSignersFromDatabase(key);
+        signers = previousEntry->data.account().signers;
+        assert(std::adjacent_find(signers.begin(), signers.end(),
```

Comment:
how do we know that `previousEntry` not set implies that we can skip loading from the database? (probably a question pertinent to all entries, not just signers)